### DISCUSSION TITLE
Plugin-free language syntax

### DISCUSSION THREAD

**Discussion by bettywolfy** [Acode-Foundation/Acode#893] [Ideas]
As far as I know, you need to create a plugin to add a syntax, but could it be possible to have the option to create syntaxes without the need to create a plugin first?

That's all I have to say, nothing more :-/

(So that you don't misunderstand me, I'm saying you can create a law for the editor to "color" the code according to your syntax, example: coloring "function" red for Javascript syntax.)

**Comment by bajrangCoder** [comment]
I don't think that will be practical because you will need to interact with many APIs etc for creating language syntax

**Reply by bettywolfy** [reply]
I think changing how this works would help, but it seems unfeasible, but still, having the option to add syntax using yaml and regexs seems useful.
